pkg/signup/signup.go

```diff
@@ -40,6 +40,12 @@ type Signup struct {
 	// Company from the Identity Provider
 	Company string `json:"company"`
 	Status  Status `json:"status,omitempty"`
+	// StartDate is the date that the user's current subscription started, in ISO8601 format
+	StartDate *string `json:"start_date,omitempty"`
+	// End Date is the date that the user's current subscription will end, in ISO8601 format
+	EndDate *string `json:"end_date,omitempty"`
+	// DaysRemaining is a float representing the number of days remaining in the user's subscription
+	DaysRemaining *float64 `json:"days_remaining,omitempty"`
```

Comment:
we use camel-case all over the place, not sure why you want to use snake-case in this particular situation
```suggestion
	StartDate *string `json:"startDate,omitempty"`
	// End Date is the date that the user's current subscription will end, in ISO8601 format
	EndDate *string `json:"endDate,omitempty"`
	// DaysRemaining is a float representing the number of days remaining in the user's subscription
	DaysRemaining *float64 `json:"daysRemaining,omitempty"`
```

Reply:
apart from that, why do you want to return the fields as pointers? This is not really need, is it?

Reply:
You're right, it looks like `omitempty` will still work if a string is equal to its default value, i.e. empty.  Fixed.